ngraph/core/src/descriptor/tensor.cpp

```diff
@@ -93,7 +93,7 @@ const Shape& descriptor::Tensor::get_shape() const
 
 size_t descriptor::Tensor::size() const
```

Comment:
If I remember right the initial idea was to return the size of Tensor in bytes.

But I don't see any documentation for this method.

Reply:
Yes I see, my mistake